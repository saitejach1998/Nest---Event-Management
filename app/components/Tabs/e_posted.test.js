import React from 'react';
import User_Events from './e_posted.js';

import renderer from 'react-test-renderer';

describe('User_Events component', function() {
    it('renders a SignupCompont ', function() {
      const tree = renderer.create(
        <User_Events />
        ).toJSON();
    expect(tree).toMatchSnapshot();
    });
  });