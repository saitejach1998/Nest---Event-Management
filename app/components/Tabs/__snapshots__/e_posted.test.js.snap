// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User_Events component renders a SignupCompont  1`] = `
<RCTScrollView
  data={Array []}
  disableVirtualization={false}
  getItem={[Function]}
  getItemCount={[Function]}
  horizontal={false}
  initialNumToRender={10}
  keyExtractor={[Function]}
  maxToRenderPerBatch={10}
  numColumns={1}
  onContentSizeChange={[Function]}
  onEndReachedThreshold={2}
  onLayout={[Function]}
  onMomentumScrollEnd={[Function]}
  onRefresh={[Function]}
  onScroll={[Function]}
  onScrollBeginDrag={[Function]}
  onScrollEndDrag={[Function]}
  refreshControl={
    <RefreshControlMock
      onRefresh={[Function]}
      refreshing={true}
    />
  }
  refreshing={true}
  renderItem={[Function]}
  scrollEventThrottle={50}
  stickyHeaderIndices={Array []}
  updateCellsBatchingPeriod={50}
  viewabilityConfigCallbackPairs={Array []}
  windowSize={21}
>
  <RCTRefreshControl />
  <View />
</RCTScrollView>
`;
